charts/core-dump-handler/README.md

```diff
@@ -6,28 +6,72 @@ This helm chart is designed to deploy functionality that automatically saves cor
 
 The [Helm](https://helm.sh/) cli to run the chart
 
-An [S3](https://en.wikipedia.org/wiki/Amazon_S3) compatible object storage solution such as [IBM Cloud Object Storage](https://cloud.ibm.com/objectstorage/create)
+An [S3](https://en.wikipedia.org/wiki/Amazon_S3) compatible object storage solution.
```

Comment:
I would stress here that the ObjectStore needs to implement the _S3 Protocol_. I had a hard time explaining to our customers that they were in fact not required to run this on AWS.

Reply:
https://github.com/IBM/core-dump-handler/pull/53/commits/251fcbe85a5499a7cb7713e5da3f6a4ea058d1fe